smart_campus/app/models.py

```diff
@@ -227,9 +232,40 @@ def __str__(self):
             category=str(self.category)
         )
 
+    def get_primary_image(self):
+        """
```

Comment:
Following the [google style python docstrings guide](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) would be better.

e.g.
```python
def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.

    .. _PEP 484:
        https://www.python.org/dev/peps/pep-0484/

    """
```

Reply:
Got it, giving it a try.